docs/en/contributing-to-appium/appium-packages.md

```diff
@@ -4,7 +4,7 @@
 
 ## appium
 - runs basic checks
-  - node version (>= 4)
+  - node version (>= 5)
```

Comment:
seems surprising to me that we would only list required node version in our docs here in the list of appium packages. are there no other places?

Reply:
That was all I could find! I'll keep my eyes peeled when doing the docs work, though.